core/src/main/kotlin/utilities/json.kt

```diff
@@ -20,10 +20,11 @@ private val objectMapper = run {
 }
 
 @PublishedApi
-internal class TypeReference<T> private constructor(
+internal class TypeReference<T> @PublishedApi internal constructor(
     internal val jackson: JacksonTypeReference<T>
 ) {
     companion object {
+        @PublishedApi
         internal inline operator fun <reified T> invoke(): TypeReference<T> = TypeReference(jacksonTypeRef())
```

Comment:
I think the rules for exposing internal declarations in inline functions have become more strict.

This is called in inline functions, so it needs to be `@PublishedApi` - not sure why it didn't before, seems logical. 

And it used to call a private constructor, which is not allowed now, so the constructor needs to be both `internal` and annotated with `@PublishedApi` now